test/k8s_service_config_injection_template_test.go

```diff
@@ -67,6 +67,38 @@ func TestK8SServiceEnvVarAddsEnvVarsToPod(t *testing.T) {
 	assert.Equal(t, renderedEnvVar["DB_PORT"], "3306")
 }
 
+// Test that setting the `envVars` input value will include those environment vars
+// We test by injecting to the envVars:
+// DB_HOST: "mysql.default.svc.cluster.local"
+// DB_PORT: 3306
```

Comment:
NIT: update the comments to reflect what the test is actually doing

Reply:
You're always to fast for me!